rpm/kubelet.spec

```diff
@@ -13,15 +13,11 @@
 
 # no support for %elseif (??)
 # https://github.com/rpm-software-management/rpm/issues/311
-%if %{KUBE_SEMVER} >= %{semver 1 11 0}
```

Comment:
i am very new to this, what the issue that causing to revert back the CNI version bump?
is it make sense for k8s v1.14+ to require CNI version >= 0.7.5?

Reply:
pinning the old packages to `= 0.6.0` does not work, possibly because the MAJOR is `0` here.

if we put `>= 0.6.0` and if `kubernetes-cni 0.7.5` already exists in the package repository it's going to be used instead of 0.6.0.

all in the lines of AFAIK.

Reply:
> pinning the old packages to `= 0.6.0` does not work, possibly because the MAJOR is `0` here.
sorry, i still not fully understand the issue.
if pinning not working, then line 16-20 may could be replaced with `global CNI_VERSION 0.6.0`?

> if we put `>= 0.6.0` and if `kubernetes-cni 0.7.5` already exists in the package repository it's going to be used instead of 0.6.0.
this make sense to me

Reply:
> sorry, i still not fully understand the issue.

i don't either, with respects to the "MAJOR is 0" part.

> if pinning not working, then line 16-20 may could be replaced with global CNI_VERSION 0.6.0?

yes, it can be replaced with 0.6.0.
because `CNI_VERSION 0.5.1` is for 1.8 which is outside of the support skew. but i don't want to touch these older versions.

Reply:
got you, thx